### PR TITLE
Add NuGet package configuration details for KaitaiStruct.Runtime.CSharp.

### PR DESCRIPTION
**Pull request by Arlorean**
Changes needed to create a NuGet package from the project for .NET 4.5+.
Package can be [downloaded from here](https://www.nuget.org/packages/KaitaiStruct.Runtime.CSharp/).

### CODE REVIEW

File: kaitai_struct_runtime_csharp.csproj [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
   <PropertyGroup>
-    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Configuration Condition=" '$(Configuration)' == '' ">Release</Configuration>
     <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
     <ProjectGuid>{8339A750-C407-4CE8-8E31-51CB2EFD3A4B}</ProjectGuid>
     <OutputType>Library</OutputType>
-    <RootNamespace>kaitai_struct_runtime_csharp</RootNamespace>
-    <AssemblyName>kaitai_struct_runtime_csharp</AssemblyName>
+    <RootNamespace>KaitaiStruct</RootNamespace>
+    <AssemblyName>KaitaiStruct.Runtime.CSharp</AssemblyName>
```
Comment (GreyCat) [inline kaitai_struct_runtime_csharp.csproj]: As far as I understand:

* We use just `Kaitai` namespace everywhere in .NET => probably "root namespace" should be just "Kaitai", although I'm not sure what it affects.
* Assembly name should be probably `Kaitai.Struct.Runtime` or something like that:
  * Judging from Microsoft's recommendations, first component of the assembly name should be organization, then the project, subproject, etc.
  * There would be probably no point to add `CSharp`, as this assembly would work equally well for any .NET language (C#, C++, F#, VB.NET, etc), and it is actually not that important what it is written in originally.

**Comment by GreyCat** [comment]
Thanks for your contribution — it's been a long-awaited packaging effort for a nuget package. I've added minor note about naming — could you take a look?

**Comment by Arlorean** [comment]
I’ll update the name to `Kaitai.Struct.Runtime` but I won’t publish a new package until there’s a Kaitai NuGet.org account.

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by GreyCat** [comment]
Thanks, merging in!

**Comment by Arlorean** [comment]
Here's a build version (with the correct spelling). Unzip it and upload it to the nuget.org account you setup. Or let me know the username/password and I can do it. 

[Kaitai.Struct.Runtime.0.7.0.nupkg.zip](https://github.com/kaitai-io/kaitai_struct_csharp_runtime/files/1652823/Kaitai.Struct.Runtime.0.7.0.nupkg.zip)

**Pull request merged**